src/backend/data/models/bsd-event-attendee.js

```diff
@@ -17,6 +17,12 @@ export default function(sequelize, DataTypes) {
           as: 'person',
           constraints: false
         })
+
+        BSDEventAttendee.belongsTo(models.BSDEvent, {
```

Comment:
Not entirely necessary for this particular bit because the relation is already set up in BSDEvent, but no harm in it either.